lib/pbench/server/api/resources/query_apis/datasets/datasets_mappings.py

```diff
@@ -103,10 +103,7 @@ def _get(self, json_data: JSON, request: Request) -> Response:
 
         index = IndexMapBase.ES_INTERNAL_INDEX_NAMES[json_data["dataset_view"]]
         try:
-            index_name = index["index"]
-            template = Template.find(index_name)
-            mappings = template.mappings
-
+            mappings = IndexMapBase.get_mappings(index)
```

Comment:
Why did you make this `@staticmethod`? Granted, there's no reason it couldn't be. But either way you should be able to `self.get_mappings(index)`.

Reply:
That's because `DatasetsMappings` does not inherit from `IndexMapBase` so we can not use `self` here. `DatasetMappings` does not have a `name` as a required schema parameter which is why it can not inherit from `IndexMapBase`

Reply:
My IDE seems to prefer static methods wherever a method doesn't reference `self`.  I don't know whether they are really "better", or whether it's some sort of "best practice", or whether it's just suggesting it because it can....